### PR TITLE
2.6 homepage

### PR DESCRIPTION
**Pull request by nottrobin**
A slight iteration on #15, just to split the "Doc updates" into 2 columns.

Closes #15

### CODE REVIEW

**Comment by hatched** [comment]
Because of the fake header it's a little weird in small screens. Does vanilla contain a system for this? If not it's probably best to not 'cheat' this UI.

Screen Shot 2019-05-02 at 4 46 20 PM

**Comment by nottrobin** [comment]
@hatched yeah you're right. You're also a killjoy, but you're right. @anthonydillon how do I solve this?

**Comment by anthonydillon** [comment]
@nottrobin Yeah, remove the empty `h2` from the second column and move the "Doc updates" title to a row above the others

**Comment by nottrobin** [comment]
@anthonydillon like this?

It still ends up with a little gap in mobile view, but I guess this is the best we can do?

image

File: templates/homepage.html [hunk 1]
```diff
@@ -20,24 +20,26 @@ <h2>Getting started</h2>
         </div>
       </div>
       <hr class="is-deep">
+      <div class="u-equal-height">
```
Comment (barrymcgee) [inline templates/homepage.html]: If this only has one child, it shouldn't have an equal height util?

Comment (nottrobin) [reply templates/homepage.html]: Can it just be `<div>`? Or does it need some kind of class?

Comment (nottrobin) [reply templates/homepage.html]: Removed

**Review by barrymcgee — APPROVED** [review]

**Pull request merged**